package.json

```diff
@@ -175,7 +176,6 @@
     "react-dom": "^18.2.0",
     "sass": "^1.62.1",
     "storybook": "^7.0.2",
```

Comment:
Is this supposed to be `^7.1.x` or is it just the `@storybook/addon-designs` package that needed to be updated?

Reply:
Yea just the design package. The other updates are in the lockfile. In general, updating versions in the package.json is not needed if an `npm update` is ok.